pkgs/desktops/gnome/extensions/extensionRenames.nix

```diff
@@ -26,4 +26,16 @@
 
   # That extension is broken because of https://github.com/NixOS/nixpkgs/issues/118612
   #"flypie@schneegans.github.com" = null;
+
+  # These are overrides for the manually packaged ones (some of which got deleted – as in: replaced with
+  # some automatic packaging –, but we still need the "old" name for backwards compatibility).
+  "appindicatorsupport@rgcjonas.gmail.com" = "appindicator";
+  "unredirect@vaina.lt" = "disable-unredirect";
+  "drawOnYourScreen@abakkk.framagit.org" = "draw-on-your-screen";
+  "EasyScreenCast@iacopodeenosee.gmail.com" = "easyScreenCast";
+  "gnome-fuzzy-app-search@gnome-shell-extensions.Czarlie.gitlab.com" = "fuzzy-app-search";
```

Comment:
Why not do this in passthru as well?

Reply:
Because I only put the `extensionSlug` in passthru, and these differ from it.

Reply:
I guess we could add a comment like `extensionPortal slug differs from the attribute name so it is not used.` to the `passthru`. And have these in a separate section `These are manually packaged extensions that have different attribute name from the slug on the extension portal`.

Reply:
I just noticed/remembered: Most of these extensions are not manually packaged anymore (all except three or something), thus we can't easily override that actually.

Reply:
Yeah, hence the suggestion to add another section to `extensionRenames`. Or do you mean that the `pname`s will not be changed and thus Repology will be confused?

Reply:
I'm sorry, but I don't think I understand. Isn't this what we're already doing?

Reply:
Then I am not sure what you mean by “we can't easily override that actually” :smile_cat:

Reply:
Okay. Let's have a look at `unredirect@vaina.lt` for example. It is automatically packaged, so its `extensionPortalSlug`, `disable-unredirect-fullscreen-windows` would become its `pname`. The line `"unredirect@vaina.lt" = "disable-unredirect";` changes that to `disable-unredirect`. That name is (arguably) more sensible than the other one, but also it is the one we previously had and we need to keep it by for backwards compatibility.

Reply:
That only affects the attribute path, not the derivation name:

```
nix-repl> gnomeExtensions.disable-unredirect
«derivation /nix/store/xwhialxgr3qpfwvn4r0mbzmzzdlcsyr1-gnome-shell-extension-disable-unredirect-fullscreen-windows-3.drv»
```

Reply:
Yes, because of how the pipeline works. Is this a problem?

Reply:
It might be an issue for Repology, depending on how other repositories name the extension. But we can fix that separately since this was already an issue before this PR.